src/core/unicode.js

```diff
@@ -1630,8 +1630,29 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function keepUnifiedIdeograph(str, normalizedUnicodesMap) {
+  // a character id map to a unicode. replace it if it is not unified
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str]) {
```

Comment:
I would keep the `!== undefined` check here so we know the behavior is equal to the situation before.